ansible/mnesia_restore.yml

```diff
@@ -0,0 +1,66 @@
+---
+# Playbook used to download (archived) Mnesia backup from last host in the playbook hosts
+#
+# By default the archive destination is overwritten if already exists,
+# if archive overwrite is not intended the caller is responsible for setting correct (e.g. unique) "backup_suffix"
+#
+# Example steps to restore a mnesia backup:
+# 1. Stop the node
+# 2. Delete existing mnesia database if any
+# 3. Disable mining autostart and empty the peers list in the config
+# 4. Start the node
+# 5. Restore the backup with "mnesia:restore("/tmp/mnesia.bak", [])."
+# 6. Enable mining autostart and restore the peers list in the config
+# 7. Start the node
+
+- name: Backup, archive and download Mnesia database
+  hosts: "{{ ansible_play_hosts | last }}"
+  remote_user: aeternity
+
+  tasks:
+    - name: Include configuration variables
+      include_vars: backup/config.yml
+
+    - name: Download backup to S3 bucket latest
+      aws_s3:
+        bucket: "{{ backup_bucket }}"
+        object: "{{ backup_archive_filename_latest }}"
+        dest: /tmp/{{ backup_archive_filename_latest }}
+        mode: get
+
+    - name: Unaarchive backup
+      unarchive:
+        remote_src: yes
+        src: /tmp/{{ backup_archive_filename_latest }}
+        dest: /tmp/
+
+    - import_tasks: backup/disable_mining_and_peers.yml
+
+    - meta: flush_handlers
+
+    - name: Wait aeternity node API to boot
```

Comment:
why you don't include `tasks/health_check.yml` ?

Reply:
because this: https://github.com/aeternity/infrastructure/blob/master/ansible/deploy.yml#L227 was using this as source.

Reply:
and because VENV configuration 
```TASK [Wait aeternity node API to boot] *****************************************
Tuesday 12 March 2019  11:03:05 +0000 (0:00:00.866)       0:00:16.830 ********* 
fatal: [18.130.244.71]: FAILED! => changed=false 
  module_stderr: |-
    /bin/sh: /var/venv/bin/python: not found
  module_stdout: ''
  msg: |-
    MODULE FAILURE
    See stdout/stderr for the exact error
  rc: 127
	to retry, use: --limit @/infrastructure/ansible/mnesia_snapshot.retry```